.npmrc

```diff
@@ -1,3 +1,4 @@
 # Set a custom registry for a scope that is picked up by pnpm when resolving packages;
 # This affects the lockfile format and is here to cover this case.
 @types:registry=https://registry.yarnpkg.com
+public-hoist-pattern[]=ms
```

Comment:
comment that bazel / rules_js won't actually see this yet, it only has effect for `pnpm i`

Reply:
✅